### PR TITLE
OCPBUGS#7834: Change the network type in install-config.yaml file for baremetal

### PR DESCRIPTION
**Pull request by xenolinux**
Change the network type in install-config.yaml file for baremetal 

Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OCPBUGS-7834

Link to docs preview: https://56264--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#installation-bare-metal-agent-installer-config-yaml_preparing-to-install-with-agent-based-installer

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-bare-metal-agent-installer-config-yaml.adoc [hunk 1]
```diff
@@ -54,7 +54,7 @@ Class E CIDR range is reserved for a future use. To use the Class E CIDR range,
 ====
 +
 <7> The subnet prefix length to assign to each individual node. For example, if `hostPrefix` is set to `23`, then each node is assigned a `/23` subnet out of the given `cidr`, which allows for 510 (2^(32 - 23) - 2) pod IP addresses. If you are required to provide access to nodes from an external network, configure load balancers and routers to manage the traffic.
-<8> The cluster network plugin to install. The supported values are `OVNKubernetes` (default value) and `OpenShiftSDN`.
```
Comment (xphyr) [inline modules/installation-bare-metal-agent-installer-config-yaml.adoc:57]: I believe that the original line should remain. both OVNKubernetes and OpenShiftSDN are supported for Compact and traditional clusters. It is only if you are building a SNO cluster, that the only option is OVNKubernetes.

Comment (xenolinux) [reply modules/installation-bare-metal-agent-installer-config-yaml.adoc:57]: Addressed

**Comment by zniu1011** [comment]
Look good to me

**Comment by Amrita42** [comment]
lgtm

**Comment by mburke5678** [comment]
@xenolinux Why is this bug fix for 4.12+? I see `networkType: OpenShiftSDN` in 4.9 through 4.11 also. Should we CP back to 4.9?

**Comment by xenolinux** [comment]
> @xenolinux Why is this bug fix for 4.12+? I see `networkType: OpenShiftSDN` in 4.9 through 4.11 also. Should we CP back to 4.9?

@mburke5678 The reporter has confirmed that the changes apply to only 4.12+. Since this change is related to the agent-based installer.

**Pull request merged**